### PR TITLE
Jitpack integration

### PR DESCRIPTION
**Pull request by Namiii**
In this PR: 
- Temporarily removed the signing config to allow project to be build by jipack
- Corrected some javadoc errors that were failing the build

Once this is merged into `v3.0` branch, we can then use a `SNAPSHOT` of that branch in our app.

### CODE REVIEW

File: src/main/java/co/omise/models/Account.java [hunk 1]
```diff
@@ -21,7 +21,7 @@ public void setEmail(String email) {
     }
 
     /**
-     * The {@link RequestBuilder<Account>} class for for getting the user's Account. This class only overrides the path() method from its parent.
+     * The {@link RequestBuilder} class for for getting the user's Account. This class only overrides the path() method from its parent.
```
Comment (nuxzero) [inline src/main/java/co/omise/models/Account.java]: `for for`? 😅

File: src/main/java/co/omise/models/Charge.java [hunk 2]
```diff
@@ -242,7 +242,7 @@ public void setSource(Source source) {
     }
 
     /**
-     * The {@link RequestBuilder<Charge>} class for for creating a Charge.
+     * The {@link RequestBuilder} class for for creating a Charge.
```
Comment (nuxzero) [inline src/main/java/co/omise/models/Charge.java]: `for for`? 😅

File: src/main/java/co/omise/models/Account.java (hunk 1 above)
Comment (Namiii) [reply src/main/java/co/omise/models/Account.java]: I don't know why this keeps happening to me 😂

**Comment by Namiii** [comment]
@nuxzero Done

File: src/main/java/co/omise/requests/Requester.java [hunk 3]
```diff
@@ -15,34 +15,35 @@
...
-     * Sends the user generated {@link Request<T>} to {@link Requester} for it to be carried out
+     * Sends the user generated {@link Request} to {@link Requester} for it to be carried out
      *
      * @param <T>     the {@link Model} object type that is expected to be returned
      * @param <R>     the {@link Request} object type that is passed in from the user
-     * @param request the {@link Request<T>} user generated request
+     * @param request the {@link Request} user generated request
      * @param klass   the type of the object that the response is expected to be deserialized as
-     * @return the {@link Model} object of type <T>
+     * @return the {@link Model} object extending from Model
```
Comment (pitiphong-p) [inline src/main/java/co/omise/requests/Requester.java]: I think we can have a better description of what object that would be returned from these methods

File: build.gradle [hunk 4]
```diff
@@ -48,9 +48,10 @@ distributions {
     }
 }
 
-signing {
-    sign configurations.archives
-}
+//TODO Uncomment this before any real release
```
Comment (pitiphong-p) [inline build.gradle]: How's about `Turn the signing action back on when we'll merge this into the `master`

Comment (Namiii) [reply build.gradle]: Much better

File: src/main/java/co/omise/requests/Requester.java (hunk 3 above)
Comment (Namiii) [reply src/main/java/co/omise/requests/Requester.java]: Hmmmm let me see how I can improve this 👍

**Pull request merged**